expression/builtin_other_vec.go

```diff
@@ -75,11 +76,29 @@ func (b *builtinValuesJSONSig) vecEvalJSON(input *chunk.Chunk, result *chunk.Col
 }
 
 func (b *builtinBitCountSig) vectorized() bool {
-	return false
+	return true
 }
 
 func (b *builtinBitCountSig) vecEvalInt(input *chunk.Chunk, result *chunk.Column) error {
-	return errors.Errorf("not implemented")
+	n := input.NumRows()
+	if err := b.args[0].VecEvalInt(b.ctx, input, result); err != nil {
+		if err != nil && types.ErrOverflow.Equal(err) {
+			return err
+		}
+		return err
+	}
+	f64s := result.Int64s()
```

Comment:
We usually use the name `f64s` to represent a `[]float64`. So we'd better to use the name `i64s` for `[]int64` here.